source/Octopus.Client/OctopusAsyncClient.cs

```diff
@@ -126,9 +125,27 @@ public static async Task<IOctopusAsyncClient> Create(OctopusServerEndpoint serve
 #endif
         }
 
-        private static async Task<IOctopusAsyncClient> Create(OctopusServerEndpoint serverEndpoint, OctopusClientOptions options, bool addHandler)
+        internal static async Task<IOctopusAsyncClient> Create(OctopusServerEndpoint serverEndpoint, OctopusClientOptions options, string requestingTool)
         {
-            var client = new OctopusAsyncClient(serverEndpoint, options ?? new OctopusClientOptions(), addHandler);
+#if HTTP_CLIENT_SUPPORTS_SSL_OPTIONS
```

Comment:
This looks unrelated to the PR?

Reply:
it is related, I added a new private method above the original and also added a new defaulted parameter to the original. Git is visualising that in a confusing manner